pkg/kubectl/cmd/util/helpers.go

```diff
@@ -408,13 +414,22 @@ func AddApplyAnnotationFlags(cmd *cobra.Command) {
 	cmd.Flags().Bool(ApplyAnnotationsFlag, false, "If true, the configuration of current object will be saved in its annotation. This is useful when you want to perform kubectl apply on this object in the future.")
 }
 
+func AddApplyAnnotationVarFlags(cmd *cobra.Command, applyAnnotation *bool) {
```

Comment:
WDYT of creating "Get" versions of these flags as well so they can be used in your code above?